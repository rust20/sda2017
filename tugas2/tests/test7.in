2 2 0 1000 13 13
Second;1000;120;pedang
First;1000;1000;panah
11;3;Second
1;13;First
############S
############.
############.
############.
############.
############.
############.
############.
############.
############.
##S#########.
##.#########.
##..........M

