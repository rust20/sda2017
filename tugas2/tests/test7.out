1,13 Pahlawan yang ikut:First
11,3 Pahlawan yang ikut:Second
Akhir petualangan Gudako
Level Gudako: 1
Level pahlawan:
First: 1
Second: 1
